exercises/pangram/canonical-data.json

```diff
@@ -12,57 +12,81 @@
       ],
```

Comment:
Don't forget to change the version accordingly, the current one is (semantic versioning):
```
"version": "1.1.0",
```

Reply:
Funny you brought this up right in the moment I amended this change ^^

Reply:
Here are the guidelines governing test versioning: https://github.com/exercism/problem-specifications#test-data-versioning

Since it appears that (correct me if I'm wrong):

- there's no new property
- the existing property has not been renamed
- there are no new keys
- no key types have been changed

we don't need a major version bump here, and v1.2.0 should suffice (minor vs patch because inputs/outputs were changed).

Reply:
thanks for the guide. Changed it to 1.2